src/main/java/mate/hwdao/controllers/AddCarController.java

```diff
@@ -0,0 +1,42 @@
+package mate.hwdao.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.hwdao.Main;
+import mate.hwdao.lib.Injector;
+import mate.hwdao.model.Car;
+import mate.hwdao.model.Manufacturer;
+import mate.hwdao.service.CarService;
+import mate.hwdao.service.ManufacturerService;
+
+public class AddCarController extends HttpServlet {
+
+    private static final Injector injector =
+            Injector.getInstance(Main.class.getPackageName());
```

Comment:
I am not sure if this is wrong, but we can type in our `package` name directly into the `getInstance()` method as mentioned in the common mistakes.
![Screenshot 2021-01-20 at 19 49 11](https://user-images.githubusercontent.com/67925366/105214361-94679c80-5b58-11eb-87e1-219259a01dd5.png)

Reply:
I'm not so sure about that.